packages/app/package.json

```diff
@@ -106,6 +106,7 @@
     "react-virtualized-auto-sizer": "^1.0.2",
     "react-window": "^1.8.5",
     "redis": "^3.0.2",
+    "redux-saga": "^1.1.3",
```

Comment:
Oh yeah, probably as of my changes yesterday, hidden by app image not having rebuilt.

Reply:
Seems like it https://github.com/opstrace/opstrace/commit/f4e890bee17cb0d071eddf936a2d9525fe1496f0#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L55-L56 was this.

Dependencies management is hard... I think we can move it here safely. Let's wait for the builds.

Reply:
> I think we can move it here safely 

I guess not...

```
[2021-01-06T14:02:47Z]   Try `npm i --save-dev @types/got` if it exists or add a new declaration (.d.ts) file containing `declare module 'got';`
```
https://buildkite.com/opstrace/prs/builds/3365#133eb72f-5b47-4183-a989-234808e4f611/1061-1170